accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -151,6 +524,10 @@ Windows-specific:
 
 ### System.IO.Pipes
 
+* **AnonymousPipeServerStreamAcl**
+    - `Create(PipeDirection, HandleInheritability, Int32, PipeSecurity)`
+* **NamedPipeServerStreamAcl**
+    - `Create(String, PipeDirection, Int32, PipeTransmissionMode, PipeOptions, Int32, Int32, PipeSecurity, HandleInheritability, PipeAccessRights)`
```

Comment:
Shouldn't the containing assembly be marked Windows-specific?

Reply:
@carlossanlop @jozkee

Reply:
These APIs do compile only for Windows but the assembly is not only about ACLs, plus [the project](https://github.com/dotnet/runtime/blob/50a999dd3597a33b62a7a1a1712c849e764d50e9/src/libraries/System.IO.Pipes/src/System.IO.Pipes.csproj#L97-L100) contains many other APIs that are cross platform.

Reply:
@terrajobst these APIs are indeed Windows-only.

As @Jozkee pointed out, the source code lives in a cross-platform assembly, but the actual refs are built for Windows only. See [here](https://github.com/dotnet/runtime/blob/50a999dd3597a33b62a7a1a1712c849e764d50e9/src/libraries/System.IO.Pipes.AccessControl/ref/System.IO.Pipes.AccessControl.cs#L69-L77) and [here](https://github.com/dotnet/runtime/blob/50a999dd3597a33b62a7a1a1712c849e764d50e9/src/libraries/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj#L16-L18).

Reply:
Removed